cmd/containerboot/services.go

```diff
@@ -35,6 +35,9 @@ import (
 
 const tailscaleTunInterface = "tailscale0"
 
+// Modified using a build flag to speed up tests.
+var testSleepDuration string
```

Comment:
Sorry @irbekrm, this is exactly the opposite of a recent review comment I left, but it was the least intrusive way I could think of knocking 10s off the egress services test.